ssl/t1_lib.c

```diff
@@ -1318,7 +1318,7 @@ TICKET_RETURN tls_decrypt_ticket(SSL *s, const unsigned char *etick,
         /* Some additional consistency checks */
         if (p != sdec + slen || sess->session_id_length != 0) {
```

Comment:
A better more explicit quote:
"The behavior is undefined in the following circumstances:
...
The value of a pointer that refers to space deallocated by a call to the free or
realloc function is used (7.20.3)."